projectile.el

```diff
@@ -5125,6 +5125,13 @@ Otherwise behave as if called interactively.
     (advice-remove 'compilation-find-file #'compilation-find-file-projectile-find-compilation-buffer)
     (advice-remove 'delete-file #'delete-file-projectile-remove-from-cache))))
 
+(if (bound-and-true-p savehist-loaded)
```

Comment:
I think it'd be best to move such setup to `projectile-mode`.

Reply:
Yes sure. I updated